python/src/trezorlib/transport/udp.py

```diff
@@ -141,6 +141,7 @@ def write_chunk(self, chunk: bytes) -> None:
             raise TransportException("Unexpected data length")
         LOG.log(DUMP_PACKETS, f"sending packet: {chunk.hex()}")
         self.socket.sendall(chunk)
+        time.sleep(0.0001)
```

Comment:
why? this worked fine before without the delay

Reply:
it did for TT, but actually didn't for TR. (at least on my machine)

Reply:
it was failing during firmware upload to be specific

Reply:
That makes zero sense. I would like to look into this, but don't want to hold up the PR. Please remove this change from here, merge without it, and possibly open a separate PR for this.

Reply:
Deleted in [abc042e](https://github.com/trezor/trezor-firmware/pull/2959/commits/abc042ee3467c443719f78fb1618c224d4d9a417)

Reply:
After deleting this, I am also _sometimes_ experiencing `USB error` during firmware upload